perfscale_regerssion_ci/scripts/large_network_policy.sh

```diff
@@ -0,0 +1,49 @@
+#/!/bin/bash
+################################################
+## Auth=prubenda@redhat.com qili@redhat.com
+## Desription: Script for creating pause deployments and adding network policies
+## Polarion test case: OCP-26279 - [BZ 1752636] Networkpolicy should be applied for large namespaces
+## https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-26279
+## Bug related: https://bugzilla.redhat.com/show_bug.cgi?id=1752636
+## Cluster config: 3 master (m5.2xlarge or equivalent) with 27 worker
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/large-network-policy.yml
+################################################
+
+for i in $(oc get projects | grep large-network-policy | grep -Eo 'large-network-policy\S*');
```

Comment:
are you running kube-burner before this step? It is very confusing that the namespace creation isn't a part of this script

Reply:
@paigerube14 I want to discuss with you about how to call the custom kube-burner, from a script or making use of your existing kube-burner Jenkins job(may need some change). As we discussed yesterday, we both think script is better, which is clear and we can add all logic in a single test case entry script.